notebooks/nmf_human_heart_consensu_tutorial.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
Let's include headers and explanations in this notebook.  Make it look clear and understandable.